game/game_application.cpp

```diff
@@ -22,10 +22,14 @@ Ref<Application> CreateRootexApplication()
 Optional<String> GameApplication::getSceneNameFromCommandLine(const char* s)
 {
 	String cmdLine = s;
-	size_t found = cmdLine.find("game/assets/");
-	if (found != String::npos)
+	if (Scene::isReservedName(cmdLine))
 	{
-		return cmdLine.substr(found, cmdLine.size() - 1);
+		size_t found = cmdLine.find("game/assets/");
+		if (found != String::npos)
+		{
+			return cmdLine.substr(found, cmdLine.size() - 1);
+		}
+		return {};
 	}
 	return {};
 }
```

Comment:
Shouldn't this be `!Scene::isReservedName` ?
Also, remove redundant return statements

Reply:
Yess, its my error 
Fixed